test/K6/src/tests/maskinporten/maskinporten.js

```diff
@@ -239,11 +271,12 @@ export function postMaskinportenSchemaNotReadyTest() {
   const appid = 'appid-302';
   
   // Act
-  var res = maskinporten.postMaskinportenSchema(offeredByToken, offeredByPartyId, toOrgNumber, appid, 'orgno');
+  var res = maskinporten.postMaskinportenSchema(offeredByToken, offeredByPartyId, appid, 'urn:altinn:organizationnumber', toOrgNumber);
 
   // Assert
   var success = check(res, {
     'post MaskinportenSchema that is Not Ready - status is 400': (r) => r.status === 400,
+    'post MaskinportenSchema that is Not Ready - `One or more validation errors occurred.`': (r) => r.json('title') == 'One or more validation errors occurred.',
```

Comment:
The Title of the problem details is usually quite generic.
For the specific error message you will need to look at the array of Errors